tensorflow_io/core/python/experimental/filter_ops.py

```diff
@@ -243,3 +243,83 @@ def gabor(
         input, tf.cast(tf.math.imag(g), input.dtype), [1, 1, 1, 1], padding="VALID"
     )
     return tf.complex(real, imag)
+
+
+def prewitt(input, mode=None, constant_values=None, name=None):
+    """
+    Apply Prewitt filter to image.
+
+    Args:
+      input: A 4-D (`[N, H, W, C]`) Tensor.
+      mode: A `string`. One of "CONSTANT", "REFLECT", or "SYMMETRIC"
+        (case-insensitive). Default "CONSTANT".
+      constant_values: A `scalar`, the pad value to use in "CONSTANT"
+        padding mode. Must be same type as input. Default 0.
+      name: A name for the operation (optional).
```

Comment:
@yongtang just wanted to know how we will be using the `name` parameter in the future. (like naming the OP etc.)